core/includes/install.core.inc

```diff
@@ -1458,6 +1468,23 @@ function install_no_profile_error() {
   return st('We were unable to find any installation profiles. Installation profiles tell us what modules to enable and what schema to install in the database. A profile is necessary to continue with the installation process.');
 }
 
+/**
+ * Indicates that Backdrop has already been installed.
+ */
+function install_database_connexion_error($errors = array()) {
```

Comment:
Should be `install_database_connection_error` to match English used elsewhere in code.